### ISSUE TITLE
misc feedback for getting started doc

### ISSUE THREAD

**Issue by mishaone** [fabric8io/fabric8-online-docs#345]
Branched off from https://github.com/fabric8io/fabric8-online-docs/issues/339#issuecomment-386895808

References to 'add to space' on dashboard to start application launcher, needs to be changed to Create an Application
5.3>8b, when I click view new app it takes me to the dashboard, we need to tell user where to go next. Space dashboard has changed no longer shows the Pipelines view, we have screenshots in 5.5 which need to be updated and instructions modified to point user to the Pipelines page.
Section 5.4 could go ahead of 5.8? Seems more relevant to the next actions user will be expected to take, wdyt?
5.9 c step needs to change, we have screenshots for experimental but we talk about internal, also as of now internal no longer exists, how about a link to settings section in user guide?
5.9 second section step, click 'view openshift console'
5.12> No projects explorer now, it is straight away app nameapp-test-1 (master)

**Comment by mishaone** [comment]
I fixed the button text, added a homepage after hello world creation screen so people know what they should see. The pipelines one I had already fixed in the previous PR since I spotted it as I went through the steps. Fixed internal features and did not add a link to the user guide since there wasn't a module about what each of these levels is about (probably should be, will create an issue for later). Fixed projects explorer view as well. 

@Preeticp not sure what you mean by this:
5.9 second section step, click 'view openshift console'
